app/src/main/java/org/sil/gatherwords/room/Converters.java

```diff
@@ -0,0 +1,21 @@
+package org.sil.gatherwords.room;
+
+import android.arch.persistence.room.TypeConverter;
+
+import java.util.Date;
+
+/**
+ * Created by andrewthomas on 1/11/18.
+ */
```

Comment:
Generally not needed to leave your name; if we want to find the author, we can use git history.